clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -275,28 +275,30 @@ check_sane_control_flow()
         if (!run_checker(memrefs, false))
             return false;
     }
-    // PC discontinuity from instr to kernel_xfer event marker value.
+    // Positive test (PC discontinuity): Transition from instr to kernel_xfer event
```

Comment:
IMHO the terminology used in existing tests is confusing: we use "negative" to mean presence of PC discontinuity. I think we should either change that as well, or just conform with the existing convention for now here too.